README.md

```diff
@@ -0,0 +1,23 @@
+# Opdrachtbeschrijving
+
+In deze repository vind je een aantal opdrachten die je kunt maken om te oefenen met lists en tuples in Python.
+Deze repository bestaat uit meerdere opdrachten van oplopende moeilijkheidsgraad. 
+
+Om deze opdracht te maken kun je op Github, onder de groene knop 'Code', deze repository downloaden als zip bestand en openen in je IDE
+(PyCharm).
+
```

Comment:
Aan het einde van de inleiding mag je dan altijd zetten:

De uitwerkingen staan op de branch _uitwerkingen_ in deze repository.